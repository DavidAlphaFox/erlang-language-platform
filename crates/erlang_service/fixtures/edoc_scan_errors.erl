%% @doc This module contains an unescaped <tag>
-module(edoc_scan_errors).

%% @doc This is function 1
one() ->
    1.

%% @docc This is function two, containing an incorrect tag
two() ->
    2.
