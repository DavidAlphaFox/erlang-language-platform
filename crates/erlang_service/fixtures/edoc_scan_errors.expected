MODULE_DOC


FUNCTION_DOCS
{}

EDOC_DIAGNOSTICS
[
    DocDiagnostic {
        severity: "error",
        line: 1,
        message: "{endtag_does_not_match,{was,doc,should_have_been,tag}}",
    },
]


